nestle.py

```diff
@@ -856,16 +947,23 @@ def prior_transform(u):
         if update_interval < 1:
             raise ValueError("update_interval must be >= 1")
 
+    # Parallel evaluation.
+    if n_threads is None or n_threads == 1:
+        n_threads = 1
+        pool = FakePool()
+    else:
+        if pool is None:
+            pool = concurrent.futures.ThreadPoolExecutor(n_threads)
```

Comment:
I agree with @aarchiba here. Ditch `n_threads` in favor of `queue_size` or `queue_length`, and throw an error if no pool is supplied.

Naming bikeshed: `n_foo` or `nfoo` denotes "number of foo" in the current parameter names (e.g., `npoints`, `ndim`). We only have one queue, so `n_queue` isn't quite right. Hence `queue_size` or `queue_length`.